pkg/services/publicdashboards/service/service.go

```diff
@@ -265,6 +265,17 @@ func (pd *PublicDashboardServiceImpl) GetOrgIdByAccessToken(ctx context.Context,
 	return pd.store.GetOrgIdByAccessToken(ctx, accessToken)
 }
 
+func (pd *PublicDashboardServiceImpl) Delete(ctx context.Context, userOrgId int64, dashboardUid string, uid string) error {
+	if dashboardUid == "" || !util.IsValidShortUID(dashboardUid) {
```

Comment:
Since this come from the request, we should be validating in the api layer before we dispatch to the service

Reply:
Since the service is the one who should be in charge of business logic, I think it's better to put this into service logic because of the error to be thrown